style.css

```diff
@@ -1,825 +1,811 @@
-    width: 80px;
+  .second {
+    height: 900px;
+    min-width: 1200px;
+    display: grid;
+    grid-template-columns: 730px 488px;
+    grid-template-rows: 593px;
+    grid-column-gap: 42px;
+    padding-top: 172px;
+    justify-content: center;
   }
 
-  .header__menu-item {
-    height: 4px;
-    width: 32.7px;
-    margin-bottom: 7px;
+  .second__bag-dsk {
+    display: block;
+    margin-top: 56px;
   }
 
-  .header__logo p {
-    margin: 11px 4px;
-    color: black;
-    font-size: 33px;
-    letter-spacing: 2px;
+  .second__box {
+    width: 488px;
+    height: 556px;
   }
 
-  .text__container {
-    display: flex;
-    align-items: flex-end;
-    width: 100%;
-    height: 462px;
-    letter-spacing: 5px;
+  .second__title {
+    font-size: 40px;
+    line-height: 1;
+    padding-top: 15px;
+    margin-bottom: 13px;
   }
 
-  .text__title {
-    position: inherit;
-    align-items: flex-end;
-    font-size: 55px;
-    line-height: 57px;
-    padding-left: 36px;
-    letter-spacing: 1px;
+  .second__container {
+    width: 377px;
+    height: 444px;
   }
 
-  .vectormob {
-    position: absolute;
-    width: 43px;
-    height: 23px;
-    left: 45.5%;
-    top: 511px;
-    display: block;
+  .second__subtitle {
+    font-size: 16px;
+    margin-bottom: 62px;
   }
-}
 
-@media (max-width: 360px) {
-  .first-container {
-    padding: 22px 0;
-    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
-    background-size: 360px;
-    background-position: center;
+  .second__text {
+    font-size: 15px;
+    margin-bottom: 43px;
   }
 
-  .header__inner {
-    width: 100%;
-    height: 65px;
-    background: #fff;
-    z-index: 2;
+  .second__block-color-gray,
+  .second__block-color-pink,
+  .second__block-color-blue,
+  .second__block-color-green {
+    width: 30px;
+    height: 30px;
   }
 
-  .header__menu {
-    background: #ededed;
-    z-index: 3;
-    height: 65px;
-    width: 65px;
+  .second__block-color-pink {
+    margin-left: 11.82px;
   }
 
-  .header__menu-item {
-    height: 4px;
-    width: 27.7px;
-    margin-bottom: 7px;
+  .second__block-color-blue {
+    margin-left: 11.82px;
   }
 
-  .header__logo {
-    margin: 4px 28px;
+  .second__block-color-green {
+    margin-left: 11.82px;
   }
 
-  .header__logo p {
-    margin: 10px 0;
-    color: black;
-    font-size: 28px;
-    letter-spacing: 2px;
+  .second__block-coust {
+    font-size: 40px;
+    margin-right: 20px;
   }
 
-  .text__container {
-    display: flex;
-    align-items: flex-end;
-    width: 100%;
-    height: 381px;
-    letter-spacing: 5px;
+  .second__quantity {
+    text-decoration: none;
+    color: #000;
+    display: inline-block;
+    font-size: 15px;
+    border: 1.5px solid #000;
+    padding: 11px 35px;
+    margin-right: 9px;
+    margin-left: 53px;
   }
-
-  .text__title {
-    position: inherit;
-    align-items: flex-end;
-    font-size: 46px;
-    line-height: 49px;
-    padding-left: 28px;
-    letter-spacing: 2px;
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)